app/models/solr_document.rb

```diff
@@ -28,6 +28,11 @@ def mixed?
     fetch(Settings.FIELDS.RESOURCE_CLASS, []) == 'Other'
   end
 
+  # a list of all the available other urls for the document
+  def also_available_links
```

Comment:
I almost feel like this method should go into the component. It is relying on `searchworks_url` which makes sense as a method on the solr_document since it's pulling from a solr field. But this is an extra layer on top of that -- it's not directly related to the solr_document. Thoughts?

Reply:
makes sense -- i'll move to the component